doc/source/whatsnew/v0.9.1.rst

```diff
@@ -59,7 +59,7 @@ New features
 
         .. ipython:: python
 
-    	    df = DataFrame(np.random.randn(5, 3), columns = ['A','B','C'])
+    	    df = pd.DataFrame(np.random.randn(5, 3), columns = ['A','B','C'])
```

Comment:
Can you revert changes to this file?

Reply:
The doc build fails due to an import error